posts/JHipster-Conf-2-Hottest-JHipster-Day-of-the-Year.md

```diff
@@ -0,0 +1,59 @@
+
+This year was also different since we were happy to welcome speakers with real-life stories which make the beauty of the Open Source community even bigger. [Pia Mancini](https://twitter.com/piamancini) who is the cofounder of [Open Collective](https://opencollective.com/) gave an awesome talk on how Open Collective helps Open Source communities by being financially transparent. [Cindy Marin](https://twitter.com/laslorma) and [Ernesto Butto](https://twitter.com/poolebu) shared their really touching story on how JHipster started a positive chain of events, you can read more about it on [Medium](https://medium.com/@ernesto_31246/a-thank-you-letter-to-the-jhipster-team-7397ad8a09fb).
+
+# JHipster updates 
+
+## JHipster 6
+
+JHipster 6 was released last month with all its major components upgraded to their latest release (Spring Boot, Angular, React, JUnit 5, etc).
+Other important changes that comes with version 6:
+- JDK 11 with JDK 8 compatibility
+- Default packaging changed to Jar (make Jar not War)
+- Bootswatch theme selection
+
+## Cloud support
+
+Overall, the cloud support is improved with the latest Spring Cloud release train: Greenwhich. Helm is now supported with its own sub-generator and the Kubernetes/Istio support was also improved.
+
+## Vue.js
+
+[Vue.js](https://vuejs.org/) is the new trending framework in the Javascript front-end world, and its simplicity lets you build web applications pretty fast. I really recommend trying the official [Vue.js blueprint](https://github.com/jhipster/jhipster-vuejs), here a [blog post](https://blog.ippon.tech/creating-a-modern-web-app-using-vuejs-and-spring-boot-with-jhipster/) that explains how Vue.js is different than Angular and React. The JHipster core team is also discussing on merging the blueprint in the main project which will simplify its usage.
+
+## Project metrics
+
+With 100,000 installations per month and more than 2 million in total, JHipster popularity is having a steady grow. The [main website](https://www.jhipster.tech/) gets 90,000 unique visitors per month and the project on [Github](https://github.com/jhipster/generator-jhipster) has more than 14,000 stargazers!
+
+## Miscellaneous
+
+If you're new to the JHipster world, you can follow a serie of [free tutorials here](https://github.com/jhipster/jhipster-guides). All you need is a Google account and you will be able to play with JHipster using Google Cloud Shell.
+
+In case you did not know, the whole JHipster's artwork was updated with a new logo and a whole JHipster family. More details can be found on the [official page](https://www.jhipster.tech/artwork/).
```

Comment:
```suggestion
In case you did not know, JHipster's artwork was updated with a new logo and a whole new JHipster family. More details can be found on the [official page](https://www.jhipster.tech/artwork/).
```